e2e/app/container.ts

```diff
@@ -31,10 +31,8 @@ export default class Container {
   }
 
   async waitUntilVisible(timeout = 60000): Promise<void> {
-    await Promise.all([
-      waitWhileLoading(this.page, timeout),
-      this.page.waitForXPath(this.getXpath(), { visible: true, timeout })
-    ]);
```

Comment:
Fix a race condition bug: Loading spinner could come after `this.page.waitForXPath()` has finished.